tests/spec/fs.copyFile.spec.js

```diff
@@ -0,0 +1,52 @@
+var util = require('../lib/test-utils.js');
+var expect = require('chai').expect;
+
+describe('fs.copyFile', function(){
+  beforeEach(function(done){
+    util.setup(function() {
+      var fs = util.fs();
+      fs.writeFile('/srcfile', 'This is a src file.', function(error){
+        if(error) throw error;
+        fs.writeFile('/destfile', 'This is a dest file.', function(error){
+          if(error) throw error;
+          done();
+        });
+      });
+    });
+  });
+  afterEach(util.cleanup);
+
+  it('should be a function', function() {
+    var fs = util.fs();
+    expect(fs.copyFile).to.be.a('function');
+  });
+
+  it('should return an error if the src path does not exist', function(done){
+    var fs = util.fs();
+    var src = null;
+    var dest = 'dest.txt';
```

Comment:
nit: consistency - here you declare `dest` as a variable, but on [line 41](https://github.com/filerjs/filer/pull/481/files#diff-fef1ff9617a76f4b8c78fb20f986a75bR41) you use the filename directly in the function call. Since the destination filename is so short, I'd suggest putting it inline.

Reply:
Agree, inline this.  Also, paths in Filer must be absolute: `/dest.txt`